data/recipes.yaml

```diff
@@ -131,6 +131,14 @@
 ##                STONE                ##
 #########################################
 
+- in:
+  - [1, rock]
```

Comment:
Why not use a renewable iron though?
If you are making new robots that require lodestone, you will need iron for their construction too.